lib/megaLogger.js

```diff
@@ -80,12 +78,26 @@
     };
 
     /**
+     * Static, converts enum to text/label
+     *
+     * @private
+     * @static
+     */
+    var _intToLevel = {
+        '40': 'ERROR',
+        '30': 'WARN',
+        '20': 'INFO',
+        '10': 'LOG',
+        '0': 'DEBUG'
+    };
```

Comment:
Here i replaced the `_intToLevel()` function by an _object_, which should be much faster than traversing the `LEVELS` in every logging invocation.